domain/schema/model/sql/0012-secret.sql

```diff
@@ -70,6 +70,14 @@ CREATE TABLE secret_value_ref (
     REFERENCES secret_revision (uuid)
 );
 
+-- Deleted revisions for which content is stored externally.
+-- These rows are deleted after the external content has been deleted.
+CREATE TABLE secret_deleted_value_ref (
+    revision_uuid TEXT NOT NULL PRIMARY KEY,
+    backend_uuid TEXT NOT NULL,
```

Comment:
We need to clean up this table when the secret backend is removed with --force.

Reply:
Is the backend refCount for the revision still in the controller database when the secret revision is removed?

Reply:
Yes. This just tracks the to be deleted external revisions for a single model.